polaris.shopify.com/src/components/Subnav/Subnav.tsx

```diff
@@ -0,0 +1,94 @@
+import Link from 'next/link';
+
+import navJSON from '../../../.cache/nav.json';
+import {NavJSON, NavItem} from '../../types';
+import {className} from '../../utils/various';
+
+import styles from './Subnav.module.scss';
+import {useRouter} from 'next/router';
+import {Icon} from '@shopify/polaris';
+import * as polarisIcons from '@shopify/polaris-icons';
+
+type PolarisIcon = keyof typeof polarisIcons;
+const nav = navJSON.children as NavJSON;
+
+function Subnav() {
+  const {asPath} = useRouter();
+  const navItems = getNavItems(asPath);
+
+  if (!navItems) {
+    console.warn('No subnav items found for path:', asPath);
+    return null;
+  }
+
+  const sortedNavItems = Object.entries(navItems).sort(([_a, a], [_b, b]) => {
+    if (a.order && b.order) return a.order > b.order ? 1 : -1;
+    return 0;
+  });
+
+  return (
+    <nav>
+      <ul className={styles.Items}>
+        {sortedNavItems.map(([key, navItem]) => (
+          <Link
+            className={className(
+              styles.Item,
+              asPath === navItem.slug && styles['Item--active'],
+            )}
+            key={key}
+            href={{pathname: navItem.slug || './'}}
+          >
+            {navItem.icon ? (
+              <Icon source={polarisIcons[navItem.icon as PolarisIcon]} />
+            ) : null}
+            {navItem.title}
+          </Link>
+        ))}
+      </ul>
+    </nav>
+  );
+}
+
+function getNavItems(path: string): {[key: string]: NavItem} | undefined {
+  const paths = path.split('/').filter((segment) => segment);
+
+  const navItemPath = paths.join('.children.');
+  const currentNavItem = getObjectValue<NavItem>(nav, navItemPath);
+
+  const isOverviewPage = currentNavItem?.children !== undefined;
+
+  // Get the parent nav item if we're on a subpage
+  const parentItemPath = paths.slice(0, -1).join('.children.');
+  const parentNavItem = isOverviewPage
+    ? currentNavItem
+    : getObjectValue<NavItem>(nav, parentItemPath);
+
+  // Return if we're on a page that doesn't have a subnav
+  if (!parentNavItem) return;
+
+  return {
+    overview: {
```

Comment:
Not a blocker for merging: @tjonx @zecarlostorre will every `Overview` page be using the polaris svg icon? we'll need that svg icon exported from figma, added to the site, and set here

Reply:
<img width="93" alt="Screenshot 2023-09-18 at 10 43 28 AM" src="https://github.com/Shopify/polaris/assets/4642404/143a7d38-408d-44c4-9828-10fb07dbb727">

This icon

Reply:
Will do!